src/dndbeyond/content-scripts/character.js

```diff
@@ -726,6 +774,13 @@ function rollAction(paneClass, force_to_hit_only = false, force_damages_only = f
                 damages.push("1d8");
                 damage_types.push("Radiant");
             }
+
+            // Cleric Blessed Strikes
+            if (character.hasClassFeature("Blessed Strikes") &&
+            character.getSetting("cleric-blessed-strikes", false)) {
```

Comment:
some indentation issues here

Reply:
Blessed Strikes applies to "weapon attacks and cantrips", so it should apply on rollItem and rollSpell. I'm unsure about rollAction. Wouldn't actions be non-weapon attacks ?

Reply:
This is based on a conversation we had some time back about us deciding weapon attacks included natural weapons (which would then include fists/claws/tails/etc.)